apps/user-portal/src/locales/en/views.ts

```diff
@@ -0,0 +1,865 @@
+                        },
+                        genericError: {
+                            description: "Error occurred while retrieving the device",
+                            message: "Something went wrong"
+                        },
+                        success: {
+                            description: "The device was successfully registered and now you can use it as a second factor",
+                            message: "Your Device Registered Successfully"
+                        }
+                    }
+                }
+            },
+            smsOtp: {
+                descriptions: {
+                    hint: "You'll receive a text message containing the verification code"
+                },
+                heading: "SMS OTP",
+                notifications: {
+                    updateMobile: {
+                        error: {
+                            description: "{{description}}",
+                            message: "Error occurred while updating the mobile number"
+                        },
+                        genericError: {
+                            description: "Error occurred while updating the mobile number",
+                            message: "Something went wrong"
+                        },
+                        success: {
+                            description: "The mobile number in the user profile is updated successfully",
+                            message: "Mobile number updated successfully"
+                        }
+                    }
+                }
+            }
+        },
+        overview: {
+            widgets: {
+                accountActivity: {
+                    actionTitles: {
+                        update: "Update account security"
+                    },
+                    description: "You are currently logged in from the following device",
+                    header: "Account activity"
+                },
+                accountSecurity: {
+                    actionTitles: {
+                        update: "Update account security"
+                    },
+                    description: "Settings and recommendations to help you keep your account secure",
+                    header: "Account security"
+                },
+                accountStatus: {
+                    header: "Your account status looks good!",
+                    list: {
+                        0: "Password health",
+                        1: "Account completion",
+                        2: "Signed in activities"
+                    }
+                },
+                consentManagement: {
+                    actionTitles: {
+                        manage: "Manage consents"
+                    },
+                    description: "Control the data you want to share with applications",
+                    header: "Consent control"
+                }
+            }
+        },
+        privacy: {
```

Comment:
Lets move the privacy policy to a separate file? And not break in to much sections, as the content is long.

Reply:
Let's do this with another PR